### ISSUE TITLE
`npm run test` fails at HEAD

### ISSUE THREAD

**Issue by hegemonic** [googleapis/google-cloud-node#5725]
I cloned this repo, then ran the following commands in its root directory:

```
npm i
npm run test
```

The command returns an error, because `semistandard` reports errors in multiple files.

**Comment by sofisl** [comment]
There are no tests at the head of this repo. I'll fix the command but to run tests you need to go into each directory in packages.

**Comment by hegemonic** [comment]
Yes, understood. But in #5724, I was making a change that affected the entire repo; I saw that there was a `test` target in the root `package.json` file, so I thought it was a good idea to run `npm run test` before sending a pull request.

If you don't want or need contributors to run `npm run test` in the repo root, please consider updating that target so it just prints a message telling users to run the command in the appropriate package directory.
